queries.yaml

```diff
@@ -623,6 +623,22 @@ post_process:
       max_items: 5
   - fn: vectordatasource.transform.drop_properties_with_prefix
     params: {prefix: mz_}
+
+  - fn: vectordatasource.transform.merge_building_features
+    params:
+      source_layer: buildings
+      start_zoom: 13
+      end_zoom: 15
+      quantize:
+        13: vectordatasource.transform.quantize_height_round_nearest_10_meters
+        14: vectordatasource.transform.quantize_height_round_nearest_5_meters
+        15: vectordatasource.transform.quantize_height_round_nearest_meter
+      exclude:
```

Comment:
I always struggle with naming. I meant `exclude` to represent that if a feature had a particular property, it should not be considered for merging. And `ignore` meant to drop those particular properties before considering it. We're `remove`ing it in the sense that those properties will get dropped if we are able to merge, but not we're not absolutely removing those properties if there aren't any collisions.

The current configuration will not consider any feature that has a `name` for merging, and will drop `addr_housenumber` and `addr_street` if it is able to merge multiple features.